src/swap.rs

```diff
@@ -278,18 +120,43 @@ pub struct SwapParams {
     pub taker: Taker,
 }
 
+/// Fetch the current `Timestamp` for the a ledger.
```

Comment:
There is a no such thing as a "timestamp" for a ledger. It'd be good to clarify what is expected. For example, it could the current time in relation of the expiry functionality of the COMIT HTLCs.